test/cmd/newapp.sh

```diff
@@ -62,6 +62,7 @@ os::cmd::expect_success_and_text 'oc new-app ruby-helloworld-sample -o yaml' 'MY
 os::cmd::expect_success_and_text 'oc new-app ruby-helloworld-sample -o yaml' 'ADMIN_USERNAME'
 os::cmd::expect_success_and_text 'oc new-app ruby-helloworld-sample -o yaml' 'ADMIN_PASSWORD'
 os::cmd::expect_success_and_text 'oc new-app ruby-helloworld-sample --param MYSQL_PASSWORD=hello -o yaml' 'hello'
+os::cmd::expect_success 'oc new-app -e FOO=BAR -f examples/jenkins/jenkins-ephemeral-template.json -o json | grep FOO'
```

Comment:
OK, switched to `expect_success_and_text`.

I could not get `-o jsonpath` to work as we needed.  I'm punting on that one unless someone can enlighten me.

By introducing along with `expect_success_and_text` the use of `grep` and `wc` (saw other examples of this in `test/cmd/*.sh`), I can confirm that the env var is applied only once. 

That said, my initial assumption is that the use of `-e` should override any pre-existing env vars set in the template.  Is that correct @csrwng?  If so, the current implementation is not overriding.

I'll push the existing changes up for review momentarily and then investigate the "to override or not override" angle.